src/video/video_source_local.rs

```diff
@@ -443,7 +443,11 @@ impl VideoSourceAvailable for VideoSourceLocal {
                 device_path: camera_path.clone(),
                 typ: VideoSourceLocalType::from_str(&caps.bus),
             };
-            cameras.push(VideoSourceType::Local(source));
+            if matches!(source.typ, VideoSourceLocalType::Unknown(_)) {
+                warn!("Ignoring camera with unknown Local Source Type: {source:#?}");
+            } else {
+                cameras.push(VideoSourceType::Local(source));
+            }
```

Comment:
@patrickelectric With this filter, we will have only the supported cameras available to the front-end, do you think it is better or worse to filter them here?

All those **unusable** devices like /dev/video15 and /dev/video23 that are created with the Raspberry Pi camera (e.g. /dev/video0) will be filtered out.

Reply:
We don't need to filter since it'll appears as unknown on the REST API. 
The API should provide everything that it can's and let the decision be taken on the frontend.